lisa/features/disks.py

```diff
@@ -44,6 +44,17 @@ def get_partition_with_mount_point(self, mount_point: str) -> PartitionInfo:
     def get_raw_data_disks(self) -> List[str]:
         raise NotImplementedError
 
+    def add_data_disk(
+        self, count: int, disk_type: schema.DiskType, size_in_gb: int
```

Comment:
size_in_gb can be optional with a default value 20. I'm thinking about the disk_type even can have a default value. It helps the method is easier to use.

Reply:
I can use StandardHDD as default disk size